tdrs-backend/init.sql

```diff
@@ -3,4 +3,21 @@ GRANT CONNECT ON DATABASE $DB_NAME TO readaccess;
 GRANT USAGE ON SCHEMA public TO readaccess;
 GRANT SELECT ON ALL TABLES IN SCHEMA public TO readaccess;
 CREATE USER $GRAFANA_USER WITH PASSWORD '$GRAFANA_PASSWORD';
-GRANT readaccess TO $GRAFANA_USER;
\ No newline at end of file
+GRANT readaccess TO $GRAFANA_USER;
+
+CREATE ROLE OFA_ADMIN;
```

Comment:
The OFA admin doesn't need access to django tables, that's why we revoke those access